tickets/src/main/java/no/unit/nva/publication/ticket/create/CreateTicketHandler.java

```diff
@@ -70,14 +67,11 @@ protected Void processInput(TicketDto input, RequestInfo requestInfo, Context co
         var ticketType = input.ticketType();
         var ticket = persistTicket(TicketEntry.requestNewTicket(publication, ticketType));
         var customer = requestInfo.getCurrentCustomer();
-
-        if (registratorPublishesMetadataAndFiles(ticketType, customer)) {
-            updateStatusToApproved(ticket);
-            publishPublication(publication);
-        } else {
-            var ticketLocation = createTicketLocation(publicationIdentifier, ticket);
-            addAdditionalHeaders(() -> Map.of(LOCATION_HEADER, ticketLocation));
+        if (PublishingRequestCase.class.equals(ticketType)) {
+            publishingRequestResolver.resolve(ticket, publication, customer);
```

Comment:
Synes vi sender for mange variabler inn i resolve metoden, alternativet er å TicketResolver som skal ta av seg alle ticket typer og sende kun ticketType inn.

Reply:
Kanskje bittelitt siden customer ikke blir mutert, men det blir ticket og publication.

Metodenavn kan også hete noe som gjør at det er tydelig at publication og ticket blir mutert?

Reply:
Samtidig går det nesten ikke an å sende færre parameter inn i metoden, da alle disse hentes fra requestInfo og input, og det er kanskje ikke optimalt å sende disse til publishingRequestResolver

Reply:
3 parametre er vel ikke så mye? Alternativet er et eller annet Request eller Context objekt som wrapper dem.